bundles/org.openhab.binding.amazonechocontrol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -634,11 +634,36 @@
 		<description>Temperature</description>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
+	<!-- Alexa.HumiditySensor -->
+	<channel-type id="relativeHumidity">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<description>Humidity</description>
```

Comment:
Could a description help understanding this channel?

Reply:
I don't understand the question. This follows the pattern of existing channels.

Reply:
Label and description are the same, so description only says "Humidity" which is not really a description. Perhaps some more text could add context, for example "Relative humidity measured by the thermostat".

Reply:
Resolved as suggested.